spec/controllers/control_panel/stories_controller_spec.rb

```diff
@@ -0,0 +1,252 @@
+
+      describe " GET #index" do
+        it "renders stories#index" do
+          get :index, board_id: board.id
+          expect(response).to render_template :index
+        end
+      end
+
+      describe "GET #new" do
+        it "renders stories#new" do
+          get :new, board_id: board.id
+          expect(response).to render_template :new
+        end
+      end
+
+      describe "POST #create" do
+        let(:request) { post :create, board_id: board.id, story: { title: 'cokolwiek', estimate: 1} }
+
+        context "success" do
+          it "changes count of Stories" do
+            expect { request }.to change { Story.count }.by(1)
+          end
+
+          it "redirects properly" do
+            request
+            expect(response).to redirect_to control_panel_board_path(story.board_id)
+          end
+
+          it "gives 302 http code" do
+            request
+            expect(response).to have_http_status(302)
+          end
+
+          it "flashes message" do
+            request
+            expect(flash[:notice]).to eq I18n.t('shared.created')
+          end
+        end #context 'success'
+
+        context "failure" do
+          before do
+            allow_any_instance_of(Story).to receive(:save) { false }
+            request
+          end
+
+          it "renders stories#new" do
+            expect(response).to render_template :new
+          end
+
+          it "doesn't change Story count" do
+            expect { request }.to_not change { Story.count }
+          end
+        end #context 'failure'
+      end
+
+      describe "GET #show" do
+        it "renders stories#show" do
+          get :show, board_id: story.board_id,id: story.id
+          expect(response).to render_template :show
+        end
+      end
+
+      describe "GET #edit" do
+        it "renders stories#edit" do
+          get :edit, board_id: story.board_id,id: story.id
+          expect(response).to render_template :edit
+        end
+      end
+      describe "PUT #update" do
+        let!(:story) {create :story, board_id: board.id}
+        let!(:params) do
+          { board_id: story.board_id, id: story.id, story: { title: 'abc'} }
+        end
+        let(:request) { put :update, params }
+
+        context "success" do
+          it "changes story title" do
+            expect { request }.to change{ story.reload.title }.to('abc')
+          end
+
+          it "redirects properly" do
+            request
+            expect(response).to redirect_to control_panel_board_path(story.board_id)
+          end
+
+          it "gives 302 http code" do
+            request
+            expect(response).to have_http_status(302)
+          end
+
+          it "flashes message" do
+            request
+            expect(flash[:notice]).to eq I18n.t('shared.updated')
+          end
+        end #context 'success'
+
+        context "failure" do
+          before do
+            allow_any_instance_of(Story).to receive(:update) { false }
```

Comment:
you have to remember that `allow` is like FORCE so in effect you didn't check real behaviour.